contrib/babelfishpg_tsql/src/guc.c

```diff
@@ -1093,6 +1093,15 @@ define_custom_variables(void)
 				 PGC_USERSET,
 				 GUC_NOT_IN_SAMPLE | GUC_DISALLOW_IN_FILE | GUC_DISALLOW_IN_AUTO_FILE,
 				 NULL, NULL, NULL);
+
+	DefineCustomBoolVariable("babelfishpg_tsql.pltsql_linked_servers_enabled",
```

Comment:
pltsql is not requied in guc name. Also, more readable would be "enable_linked_servers"

Reply:
done